### PR TITLE
Update .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

**Comment by ukos-git** [comment]
the docu section should get an additional section:

https://github.com/byte-physics/igor-unit-testing-framework/blob/7fcc8d26d057790454509d2e9466e66b69809a7f/.gitlab-ci.yml#L11-L21

```yml
  only:
    changes:
      - docu/sphinx/source/*.rst
      - procedures/*.ipf
```

**Comment by t-b** [comment]
@ukos-git Thx. Will do.

**Comment by ukos-git** [comment]
Is this working as expected? I think it's the question if this is "OR" or "AND" for the tags.

https://github.com/byte-physics/igor-unit-testing-framework/blob/7fcc8d26d057790454509d2e9466e66b69809a7f/.gitlab-ci.yml#L47-L48

**Comment by t-b** [comment]
@ukos-git On second thought. Is that really a gain?

**Comment by t-b** [comment]
The tags are and. But we need a new one so that the shell jobs don't end up in docker. I've added shell as tags.

**Comment by ukos-git** [comment]
> But we need a new one so that the shell jobs don't end up in docker. I've added shell as tags.

sounds more like "or" to me.

**Comment by t-b** [comment]
@ukos-git Ready for review.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -20,11 +20,51 @@ documentation:
...
+    paths:
+      - tests/JU_*.xml
+      - screenshot/*.png
+
+testing-v8:
+  stage: build
+  tags:
+    - macosx, docker
+  image:
+    name: ${CI_REGISTRY}/internal/docker-igorpro:v8
```
Comment (ukos-git) [inline .gitlab-ci.yml]: Currently `stable` is the same as `v8` (Igor Pro 8.02) but `stable` will automatically increase to the latest official version available.

Comment (ukos-git) [reply .gitlab-ci.yml]: We could increase v8 to reflect the same status as `stable` in the registry. That would probably be the way to go.

Comment (t-b) [reply .gitlab-ci.yml]: Once we have IP9 I want to have an additional job for IP9.

**Pull request merged**